src/util-ebpf.c

```diff
@@ -514,11 +514,13 @@ static bool EBPFCreateFlowForKey(struct flows_stats *flowstats, FlowKey *flow_ke
      * server then if we already have something in to server to client. We need
      * these numbers as we will use it to see if we have new traffic coming
      * on the flow */
-    FlowCounters *fc = FlowGetStorageById(f, GetFlowBypassCounterID());
+    FlowBypassInfo *fc = FlowGetStorageById(f, GetFlowBypassInfoID());
     if (fc == NULL) {
-        fc = SCCalloc(sizeof(FlowCounters), 1);
+        fc = SCCalloc(sizeof(FlowBypassInfo), 1);
         if (fc) {
-            FlowSetStorageById(f, GetFlowBypassCounterID(), fc);
+            FlowSetStorageById(f, GetFlowBypassInfoID(), fc);
+            fc->BypassUpdate = EBPFBypassUpdate;
```

Comment:
why is this stored in the flow? It seems like something that will be the same for all bypassed flows, so would fit better in the flow-manager ctx itself somehow

Reply:
I was thinking about mixed capture setup. We may end up with different callback functions.

Reply:
Ok. Just feels like spending 16 bytes per flow for that use case is a bit of a heavy price.